python/main.py

```diff
@@ -9,16 +10,24 @@ def exec_statement(conn, stmt):
             row = cur.fetchone()
             conn.commit()
             if row: print(row[0])
-    except psycopg2.ProgrammingError:
+    except ProgrammingError:
         return
 
 
 def main():
 
     # Connect to CockroachDB
-    connection = psycopg2.connect(dsn=os.environ["DATABASE_URL"], application_name="$ docs_quickstart_python")
+    try:
```

Comment:
Could this try/catch block be skipped, as in the previous version of the code? The user would get a stack trace but that seems okay to me.

Reply:
Yup, removed.